docs/reference/search/search-template.asciidoc

```diff
@@ -298,3 +298,75 @@ GET /_search/template
 }
 ------------------------------------------
 <1> Name of the the query template stored in the `.scripts` index.
+
+[float]
+==== Validating templates
+
+A template can be validated with given parameters using
+
+[source,js]
```

Comment:
Can we qualify what we mean by valid/invalid?
Legal JSON?
Legal search syntax?
I'm guessing the former.